packages/action-menu/src/ActionMenu.ts

```diff
@@ -63,14 +63,29 @@ export class ActionMenu extends ObserveSlotText(DropdownBase, 'label') {
 
     protected updated(changedProperties: PropertyValues): void {
         super.updated(changedProperties);
-        if (changedProperties.has('open')) {
-            this.selected = this.open;
-        }
         if (changedProperties.has('quiet')) {
             this.quiet = true;
         }
         if (changedProperties.has('invalid')) {
             this.invalid = false;
         }
+        if (changedProperties.has('open')) {
+            this.manageMenuItems();
+        }
+    }
+
+    private async manageMenuItems(): Promise<void> {
+        if (this.selectable || !this.optionsMenu) {
+            return;
+        }
+        if (this.optionsMenu.menuItems.length) {
+            this.optionsMenu.menuItems.forEach((el) => (el.selected = false));
```

Comment:
Is this needed because `DropdownBase` causes this to get selected by default? To my comment on the issue, it might be nicer if the selected states were completely untouched if `selectable` isn't set.